### ISSUE TITLE
Runtime error when adding CRUD app

### ISSUE THREAD

**Issue by glucz** [TiagoSilvaPereira/vemto-issues#318]
I created a simple app with 3 models. This error came up when I tried to add the second CRUD.

```
TypeError: Cannot read property 'type' of undefined
    at Function.getCorrectInputFromFieldAndCrud (app://./js/chunk-common.d5e10964.js:1:64716)
    at Proxy.addInputToCrud (app://./js/chunk-common.d5e10964.js:1:24879)
    at app://./js/chunk-common.d5e10964.js:1:33662
    at Array.forEach (<anonymous>)
    at Proxy.addInputsFromModel (app://./js/chunk-common.d5e10964.js:1:33649)
    at Function.fromModel (app://./js/chunk-common.d5e10964.js:1:32986)
    at o.create (app://./js/project_apps.c4309195.js:1:12235)
    at app://./js/project_apps.c4309195.js:1:12207
```

---

vemto 1.1.4
Electron 8.5.5
win32 10.0.19044
Locale: hu

**Comment by TiagoSilvaPereira** [comment]
Hi @glucz please check #283  for a workaround... I"ll fix it soon

**Comment by TiagoSilvaPereira** [comment]
Fixed on 1.1.5
